.github/workflows/pipeline.yml

```diff
@@ -27,18 +27,26 @@ jobs:
     name: node build
     uses: ministryofjustice/hmpps-github-actions/.github/workflows/node_build.yml@v1 # WORKFLOW_VERSION
     secrets: inherit
+    with:
+      node_version: 22.11
```

Comment:
i'd suggest simply `22` to not have to keep changing this number

Reply:
yes, makes sense.. 
I'll probably remove this file and include the fix in a separate change